src/services/pcn-iptables/src/defines.h

```diff
@@ -124,7 +124,7 @@ const uint8_t DSTPORT = 4;
 // apply Horus mitigator optimization if there are at least # rules
 // matching the pattern, at ruleset begin
 const uint8_t MIN_RULE_SIZE_FOR_HORUS = 1;
-const uint8_t MAX_RULE_SIZE_FOR_HORUS = 2048;
+const uint32_t MAX_RULE_SIZE_FOR_HORUS = 2048;
```

Comment:
@mbertrone Looks strange to me that the previous line is still uint8_t. If nothing is affected, I would propose to move it to uint32_t as well, as this looks more coherent and it doesn't raise questions.

Reply:
Agree, having `uint32_t` is more coherent.
I've force-pushed this change.